src/components/AppHeader/AppHeader.module.css

```
@@ -57,7 +57,7 @@
   display: block;
 }
 
-:global(.app-container[data-layout-tier="mobile"]) .app-header-brand-icon :global(.brand-mark) {
+:global([data-layout-tier="mobile"]) .app-header-brand-icon :global(.brand-mark) {
```

Stylelint (17.12.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)